docs/developer/respond-to-issue-creation.md

```diff
@@ -0,0 +1,24 @@
+The following example notifies any user who raises an issue with
+insufficient detail in the body, via a direct message in Slack, and
+provides them with a helpful link to the issue. Note that we make use
+of the person available via the `openedBy` field:
+
+```typescript
+export async function requestDescription(inv: NewIssueInvocation) {
+    if (!inv.issue.body || inv.issue.body.length < 10) {
+        await inv.context.messageClient.addressUsers(
+            `Please add a description for new issue ${inv.issue.number}: _${inv.issue.title}_: ${inv.id.url}/issues/${inv.issue.number}`,
+            inv.issue.openedBy.person.chatId.screenName);
+    }
+}
+```
+
+This is registed with a `SoftwareDeliveryMachine` instance as follows:
+
+```typescript
+sdm.addNewIssueListeners(requestDescription)
+```
+
+Using the `credentials` on the `NewIssueInvocation`, you can easily
```

Comment:
personal preference: can we not say "easily"
like ever (it makes people feel dumb sometimes when they don't already know how to do the thing)
including here, because there's more to using the GitHub API than having credentials. "you can use the GitHub API" is accurate.

Reply:
Totally agree. 👍 (This was also just a direct cut/copy of [what existed](https://github.com/atomist/docs/pull/286/files#diff-cacc5a8626a2cc71344704d473c67b79L268)).